src/main/java/bio/terra/grammar/google/BigQueryVisitor.java

```diff
@@ -16,6 +17,12 @@ public BigQueryVisitor(Map<String, DatasetModel> datasetMap) {
   }
 
   public String generateAlias(String datasetName, String tableName) {
```

Comment:
This method doesn't need any state in `BigQueryVisitor` so it can be changed to `static`
```suggestion
  public static String generateAlias(String datasetName, String tableName) {
```